Microsoft.Toolkit.Parsers/Markdown/Enums/InlineParseMethod.cs

```diff
@@ -0,0 +1,97 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Parsers.Markdown.Enums
+{
+    internal enum InlineParseMethod
```

Comment:
How is this different than `MarkdownInlineType`?

Reply:
`InlineParseMethod` is used for identifying Inlines with TripChars, the other is for structure identification.
An inline type might have no `MarkdownInlineType`, private Inlines like BoldItalicInline which converts itself into a BoldInline, it is a workaround.

This stuff has always been here, I might clean it up / remove it in a new PR (Like with #1767 where I will likely remove the enums altogether).